modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -22,27 +22,20 @@ namespace LORIS\create_timepoint;
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris-Trunk
  */
-
 class Create_Timepoint extends \NDB_Form
 {
-    /**
-     * This timepoint's candidate
-     *
-     * @var \Candidate
-     */
-    protected $candidate;
 
-    /**
-     * The subprojectID being created
-     */
-    var $subprojectID;
+    public $skipTemplate = true; // stops from looking for a smarty template
+    public $AjaxModule   = true;
+    protected $candidate;
 
     /**
-     * Check user permissions
+     * Determine whether the user has permission to view this page
      *
      * @param \User $user The user whose access is being checked
      *
      * @return bool
+     * @throws \LorisException
```

Comment:
is the LorisException thrown from the hasPermission function?

Reply:
@zaliqarosli my IDE suggests it and I just allowed it to be autogenerated in the comment because otherwise my IDE shows ugly squiggle colours around the comment. Do you want me to remove it? I believe you're right and it's from the hasPermission function.

Reply:
I'm not sure. I just don't see a "throw LorisException" in the code so I was wondering if it actually does throw a LorisException but i'm sure your IDE is trustworthy haha